src/server/registry/handler.go

```diff
@@ -32,6 +27,9 @@ import (
 	"github.com/goharbor/harbor/src/server/registry/manifest"
 	"github.com/goharbor/harbor/src/server/registry/tag"
 	"github.com/gorilla/mux"
+	"net/http"
```

Comment:
This will be removed later right?

Reply:
yes, all handers will be bound to beego router direclty